dev/tasks/macros.jinja

```diff
@@ -221,3 +222,35 @@ on:
         cp ${formula} $(brew --repository homebrew/core)/Formula/
       done
 {% endmacro %}
+
+{%- macro change_r_pkg_version(date = '$(date +%Y%m%d)') -%}
+  - name: Modify version
+    shell: bash
+    run: |
+      cd arrow/r
+      sed -i.bak -E -e \
+        's/(^Version: [0-9]+\.[0-9]+\.[0-9]+).*$/\1.'"{{ date }}"'/' \
+        DESCRIPTION
+      head DESCRIPTION
+      rm -f DESCRIPTION.bak
+      cp ../dev/tasks/homebrew-formulae/autobrew/apache-arrow.rb tools/apache-arrow.rb
+      
+      # Pin the git commit in the formula to match
+      cd tools
+      sed -i.bak -E -e 's/arrow.git"$/arrow.git", :revision => "'"{{ arrow.head }}"'"/' apache-arrow.rb
+      rm -f apache-arrow.rb.bak
+{% endmacro %}
+
+{%- macro test_r_src_pkg() -%}
+  source("https://raw.githubusercontent.com/apache/arrow/master/ci/etc/rprofile")
+  options(arrow.dev_repo = "https://nightly.wujciak.de/r")
```

Comment:
This repo URL should be set outside of here (either as a macro or some other global setting). And it should not be your personal server.

Reply:
Yes, this was of course only for development/testing and would be removed or replaced before merging (depending on when we want to change the defaults in `*libs.R` and `install-arrow.R`).